src/platforms/shared/bidders/bidders-state.setup.ts

```diff
@@ -25,6 +25,7 @@ export class BiddersStateSetup implements DiProcess {
 		verizon: 'icPrebidVerizon',
 		yahoossp: 'icPrebidYahooSsp',
 	};
+	private notCoppaCompliantBidders: string[] = ['kargo', 'verizon'];
```

Comment:
```suggestion
	private notCoppaCompliantBidders: Array<keyof typeof this.prebidBidders> = ['kargo', 'verizon'];
```
it will make sure that you use only keys from `prebidBidders` object